### PR TITLE
Update 04.model-complexity.md

### PR DESCRIPTION
**Pull request by jaybee84**
### Manuscript checklist

Unless otherwise noted above, this PR will be considered ready for review when all four items have been checked.

- [ ] All changes to text follow "one sentence per line" [[Manubot instructions](https://github.com/jaybee84/ml-in-rd/blob/draft-branch/USAGE.md#manubot-markdown)]
- [ ] All citations follow the [Manubot citation instructions](https://github.com/jaybee84/ml-in-rd/blob/draft-branch/USAGE.md#citations)
- [ ] All changes or additions to tables follow the [Manubot table instructions](https://github.com/jaybee84/ml-in-rd/blob/draft-branch/USAGE.md#tables)
- [ ] All figures follow the [Manubot figure instructions](https://github.com/jaybee84/ml-in-rd/blob/draft-branch/USAGE.md#figures)
- [ ] All terms that need to be added to glossary have been added to #50 
- [ ] There are no new spelling errors identified by the Manubot spellchecker

### CODE REVIEW

**Comment by jaybee84** [comment]
not sure why appveyor is failing on this :(

**Comment by jaybee84** [comment]
any ideas ? @allaway @jaclyn-taroni

**Comment by allaway** [comment]
Looking at the build it seems that appveyor is having trouble with a dependency (https://ci.appveyor.com/project/jaybee84/ml-in-rd/builds/36751052)

Maybe some service is down at the moment??

**Comment by jaybee84** [comment]
Could be. Maybe something to do with the mirror sync?

>Failed to fetch https://apt.llvm.org/bionic/dists/llvm-toolchain-bionic-11/main/binary-amd64/Packages.gz  File has unexpected size (8753 != 8762). Mirror sync in progress? [IP: 199.232.198.49 443]
   Hashes of expected file:
    - Filesize:8762 [weak]
    - SHA256:06bae9b3c428cb458d55edf13dda71a5e45f0ea218ad5f4b528204ab4c6b1687
    - SHA1:851d415a6eb68cd20e9d580012e8a9a270935f86 [weak]
    - MD5Sum:f8f9adb4db3369b27ea10b5c34a5a272 [weak]
   Release file created at: Wed, 09 Dec 2020 08:00:02 +0000
E: Some index files failed to download. They have been ignored, or old ones used instead.

Ok. maybe I will try again later :)

File: content/04.model-complexity.md [hunk 1]
```diff
@@ -25,4 +25,4 @@ Hybrid applications of LASSO have also been tested in rare variant discovery, in
 In immune cell signature discovery, elastic-net regression has been used to reduce the feature space and was found to outperform other regression approaches [@doi:10.1111/j.1467-9868.2005.00503.x; @doi:10.1016/j.compbiomed.2015.10.008; @doi:10.1186/1471-2105-14-198 ; @doi:10.1186/s12859-019-2994-z].
```
Comment (allaway) [inline content/04.model-complexity.md]:
```suggestion
In immune cell signature discovery, elastic-net regression has been used to reduce the feature space and was found to outperform other regression approaches [@doi:10.1111/j.1467-9868.2005.00503.x; @doi:10.1016/j.compbiomed.2015.10.008; @doi:10.1186/1471-2105-14-198; @doi:10.1186/s12859-019-2994-z]. 
```

**Pull request merged**